new/LIT6809MEMiracleSickMonk.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6809MEMiracleSickMonk" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracle of ʾEwosṭatewos on a sick monk</title>
```

Comment:
lang="en"

Reply:
I would also extend the title to make it easier identifiable (a sick monk from Dabra Māryām and ʾAbuna ʾAbsādi)

Reply:
Done.